force-app/main/default/classes/RD2_ChangeHistoryView.cls

```diff
@@ -0,0 +1,9 @@
+public with sharing class RD2_ChangeHistoryView {
```

Comment:
Do we need a `ChangeHistoryView` and `ChangeView` class? Still thinking through it. In this case I'm not sure the extra specificity adds to the clarity since it's not readily apparent what the difference would be between the two classes.

Reply:
`RD2_ChangeHistoryView` is the top-level message class that contains the whole View for the app. Each `RD2_ChangeView` in this class represents one `RecurringDonationChangeHistory__c` record. We needed a spot to house flags for whether or not the RD2 Change History Setting is enabled, and whether or not to show the "View More" link.